packages/platform-android/native_modules.gradle

```diff
@@ -183,7 +183,7 @@ class ReactNativeModules {
 
     def cmdProcess
     def root = getReactNativeProjectRoot()
-    def command = "node ./node_modules/react-native/cli.js config"
+    def command = "node ${root.toString()}/node_modules/react-native/cli.js config"
```

Comment:
This is called on line 190 like:
```groovy
cmdProcess = Runtime.getRuntime().exec(command, null, root)
```
So I wonder, why does it change anything, since we call the command from that root directory?

Reply:
In a monorepo setup we have a shared node_modules between apps, so we call the build command from within each android app with some custom task script. This fix makes it work for those custom situations as well as the conventional situation.

Reply:
Ok, but that still doesn't address my question. I'd expect `Runtime.getRuntime` ran from the root to have `.` being a root. cc @Salakar

Reply:
Hmm I'm confused by this a bit, the third of exec is already specifying where to run the command - in the specified root dir?

Reply:
I see what you guys mean ... I just added some log and it gives the path to the `current android folder with ../` 

I tested again removing both `node` & `${root.String()}` and it didn't give the error anymore. I am not an android dev so the gradle syntax is a bit strange to me. 

Is there a way to pass special settings from the `apply` command in `settings.gradle` to let this script know where to find `node_modules`?

Reply:
Just to clarify, I thinkI got a false positive when I first changed the line to:

`def command = "${root.toString()}/node_modules/react-native/cli.js config"`
That was because I removed `node`, not because I added `${root.toString()}` 
I later added `node` by mistake when changing to make the PR without testing. Then just tested again with the last change 
`def command = "node ${root.toString()}/node_modules/react-native/cli.js config"` and I was still getting the error.

Sorry about the confusion.